src/main/java/mission1/step4/domain/car/CarEntry.java

```diff
@@ -20,26 +15,14 @@ public void regist(Car car) {
         carEntry.add(car);
     }
 
-    public Car getCar(int idx) {
-        if (carEntry.size() < idx) {
-            throw new IllegalArgumentException("엔트리에 등록된 차량의 숫자보다 높은 인덱스를 인자값 으로 허용할 수 없습니다.");
-        }
-
-        return carEntry.get(idx);
-    }
-
-    public void move(int randomValue) {
-        carEntry.forEach((car) -> {
-            car.move(randomValue);
-            if (car.getProgress() > topProgress) {
-                topProgress = car.getProgress();
-            }
-        });
-    }
-
     public List<Car> getRaceWinner() {
-        return  carEntry.stream()
-                        .filter((car) -> car.getProgress() >= topProgress)
+        int topProgress = carEntry.stream()
+                .map(Car::getProgress)
+                .max((progress1, progress2) -> progress1 - progress2)
```

Comment:
```suggestion
                .mapToInt(Car::getProgress)
                .max()
```
기본형 특화 스트림을 활용해보세요 :)